package.json

```diff
@@ -223,6 +223,7 @@
     "flow-bin": "^0.86.0",
     "jest": "24.0.0-alpha.6",
     "jest-junit": "5.2.0",
+    "jsc-android": "latest",
```

Comment:
maybe keep it simple, not introduce jsc in this pr.

Reply:
By clearing old JSC is there anything else except 64-bit libicu_common.so and libjsc.so files checked into this package?